pkg/localmetrics/localmetrics.go

```diff
@@ -83,9 +87,19 @@ var (
 	}
 )
 
-// UpdateAWSMetrics updates all AWS related metrics
-func UpdateAWSMetrics(totalAccounts int) {
-	MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(totalAccounts))
+// UpdateAWSMetrics updates the total AWS Accounts metric every N hours
+func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
+	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+		SecretName: account.AwsSecretName,
+		NameSpace:  awsv1alpha1.AccountCrNamespace,
+		AwsRegion:  "us-east-1",
+	})
+
+	d := time.Duration(hour) * time.Minute
+	for range time.Tick(d) {
+		accountTotal, _ := account.TotalAwsAccounts(awsClient)
```

Comment:
The err should be caught and logged if not nil. This means the metric will only be updated if there is no err.